packages/suite-desktop-core/webpack/core.rspack.config.ts

```diff
@@ -0,0 +1,187 @@
+
+            return '[name].js';
+        },
+        path: dist,
+        publicPath: './',
+        library: { type: 'umd' },
+    },
+    externals: [
+        ...dependencies,
+        ...devDependencies,
+        'bufferutil', // optional dependency of ws lib
+        'utf-8-validate', // optional dependency of ws lib
+    ],
+    externalsType: 'commonjs2',
+    module: {
+        rules: [
+            {
+                test: /\.(js|ts)$/,
+                exclude: [/[\\/]node_modules[\\/]/],
+                loader: 'builtin:swc-loader',
+                options: {
+                    jsc: {
+                        parser: {
+                            syntax: 'typescript',
+                        },
+                        externalHelpers: true,
+                    },
+                    env: {
+                        // electron 32 uses node 20 https://www.electronjs.org/docs/latest/tutorial/electron-timelines
+                        targets: 'node 20',
+                    },
+                } satisfies SwcLoaderOptions,
+            },
+        ],
+    },
+    resolve: {
+        modules: ['node_modules'],
+        mainFields: ['module', 'main'],
+        extensions: ['.ts', '.js'],
+        alias: {
+            '@emurgo/cardano-serialization-lib-nodejs': '@emurgo/cardano-serialization-lib-browser',
+            '@trezor/connect$': '@trezor/connect/src/index', // alternative for "module": "src/index" in connect's package.json
+            // optionally required subdependency of systeminformation lib is mocked as undefined
+            'osx-temperature-sensor': false,
+        },
+    },
+    performance: {
+        hints: false,
+    },
+    optimization: {
+        splitChunks: {
+            // By default, each entry point file would map to one output file, and those files would have a lot of duplicated code between them
+            // With this option, Rspack splits as much common code as possible into separate files to be shared
+            chunks: 'all',
+            // To avoid lots of chunk files with numerical names, they are named (and thus grouped) based on which output file shares them
+            // Rspack type declaration is missing the 2nd argument https://github.com/web-infra-dev/rspack/pull/8707
+            // @ts-expect-error
+            name: (_module, chunks: Chunk[]) => {
+                return chunks.length === 1
+                    ? chunks[0].name
+                    : `shared/${chunks.map((item: any) => item.name.split('/').pop()).join('~')}`;
+            },
+        },
+        minimizer: [
+            new rspack.SwcJsMinimizerRspackPlugin({
+                extractComments: false,
+                minimizerOptions: {
+                    format: {
+                        comments: false,
+                    },
+                },
+            }),
+            new rspack.LightningCssMinimizerRspackPlugin(),
```

Comment:
At first I thought that Rspack is slower than I expected, taking 15 s for production build, though development build only ~700ms.
Turns out, TerserPlugin is really slow! WIth the built in minimizer plugin prod build takes only 1.70 s :tada: